packages/expo/cli/register/registerAsync.ts

```diff
@@ -0,0 +1,28 @@
+import openBrowserAsync from 'better-opn';
+
+import { CI } from '../utils/env';
+import { CommandError } from '../utils/errors';
+import { ora } from '../utils/ora';
+
+export async function registerAsync() {
+  const REGISTRATION_URL = `https://expo.dev/signup`;
+  if (CI) {
+    throw new CommandError(
+      'NON_INTERACTIVE',
+      `Cannot register an account in CI. Register an account at: ${REGISTRATION_URL}`
+    );
+  }
```

Comment:
IMO it would be better to suggest creating a bot user for your organization or using a personal access token, documented here: https://docs.expo.dev/accounts/programmatic-access/

Reply:
I think matching [this](https://github.com/expo/expo/pull/16087/files#diff-21fde67c59cfa3d31ff4332a0c53e83b7b32cd425d12bec562a2a100a4b2ae2fR49-R51) would be great.
```js
{
  nonInteractiveHelp: `Use the EXPO_TOKEN environment variable to authenticate in CI (${learnMore(
    'https://docs.expo.dev/accounts/programmatic-access/'
  )})`,
 }
```